modules/administration/pages/custom-ssl.adoc

```diff
@@ -1,97 +1,195 @@
+Execute the [command]``yast2 susemanagersetup`` command from the same shell you exported the environment variables from.
 ====
 
-.Intermediate Certificates
-[TIP]
-====
-In case you want to use CAs with intermediate certificates, merge the intermediate and root CA certificates into one file.
-It is important that the intermediate certificate comes first within the combined file.
-====
 
+== Custom Certificates for New Proxy Installations
+
+By default, {productname} Proxy uses a self-signed certificate.
+After you have completed the initial setup, you can replace the default certificate with a custom certificate.
+
+.Procedure: Installing Custom Certificates on a New {productname} Proxy
 
+. Install the {productname} Proxy according to the instructions in xref:installation:install-intro.adoc[].
+. Complete the initial setup according to xref:installation:proxy-setup.adoc[].
+. At the command prompt, run:
++
+----
+configure-proxy.sh
+----
+. At the ``Do you want to import existing certificates?`` prompt, type kbd:[y]
+. Follow the prompts to complete setup.
 
 
-[[bp.cert.custom.setup]]
-== Setup
 
+== Re-Create Existing Certificates
 
-After completing {yast}
-firstboot procedures, export your current environment variables and point them to the correct SSL files to be imported.
-Running these commands will make the default certificate obsolete after executing the [command]``yast2 susemanagersetup`` command.
-For more information on {yast}
- firstboot, see https://www.suse.com/documentation/suse-manager-3/singlehtml/suse_manager21/book_susemanager_install/book_susemanager_install.html#sec.manager.inst.setup.
-[[bp.cert.custom.setup.proc]]
+If your existing custom certificates have expired or stopped working for any reason, you can generate a new certificate from the existing CA.
 
+.Procedure: Re-Creating an Existing Certificate
 
-[[bp.cert.custom.setup.proc.export]]
-. Export the environment variables and point to the SSL files to be imported:
+. On the {productname} Server, at the command prompt, regenerate the server certificate:
 +
-
 ----
-export CA_CERT=`path_to_CA_certificate_file`export SERVER_KEY=`path_to_web_server_key`export SERVER_CERT=`path_to_web_server_certificate`
+rhn-ssl-tool --gen-server --dir="/root/ssl-build" --set-country="COUNTRY" \
+--set-state="STATE" --set-city="CITY" --set-org="ORGANIZATION" \
+--set-org-unit="ORGANIZATION UNIT" --set-email="name@example.com" \
+--set-hostname="susemanager.example.top" --set-cname="example.com"
 ----
-. Execute {productname} setup with
+Ensure that the [systemitem]``set-cname`` parameter is the fully-qualified domain name of your {productname} Server.
+You can use the the [systemitem]``set-cname`` parameter multiple times if you require multiple aliases.
+. Install the RPM that contains the newly generated certificate:
 +
-
 ----
-yast2 susemanagersetup
+rpm -Uhv /root/ssl-build/lnx0259a/rhn-org-httpd-ssl-key-pair-lnx0259a-1.0-2.noarch.rpm
```

Comment:
The version/release number will be increased with every time you re-create the certificates.
Not sure if we need to tell that to our customers that they need to look for the latest version of the RPM?